### ISSUE TITLE
Unable to Install

### ISSUE THREAD

**Issue by strykeanjel** [Jalle19/xbmc-video-server#280]
Performing Fresh install of XMBC-Video-Server in XAMPP on windows but unable to complete the install on the "composer install"  command.  Looks like one of the repositories used to build has been removed

output Below:
```
====================================
USER@WEBSERVER c:\xampp\htdocs\xbmc-video-server
# composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Package operations: 26 installs, 0 updates, 0 removals
  - Installing behat/transliterator (v1.0.1): Loading from cache
  - Installing crisu83/yii-consoletools (1.0.1): Loading from cache
  - Installing crisu83/yiistrap (1.2.3): Loading from cache
  - Installing gavroche/browser (v2.2): Loading from cache
  - Installing hautelook/phpass (0.3.4): Loading from cache
  - Installing hoa/core (2.15.09.03): Loading from cache
  - Installing hoa/stream (0.15.08.28): Loading from cache
  - Installing hoa/socket (0.15.09.08): Loading from cache
  - Installing hoa/http (0.15.09.08): Loading from cache
  - Installing hoa/websocket (2.15.08.05): Loading from cache
  - Installing imagine/imagine (0.6.2): Loading from cache
  - Installing xrstf/ip-utils (v1.1.0): Loading from cache
  - Installing jalle19/php-whitelist-check (1.0.4): Loading from cache
  - Installing zendframework/zend-stdlib (2.2.10): Loading from cache
  - Installing zendframework/zend-validator (2.2.10): Loading from cache
  - Installing zendframework/zend-escaper (2.2.10): Loading from cache
  - Installing zendframework/zend-uri (2.2.10): Loading from cache
  - Installing zendframework/zend-loader (2.2.10): Loading from cache
  - Installing zendframework/zend-http (2.2.10): Loading from cache
  - Installing jalle19/simple-json-rpc-client (1.0.9): Loading from cache
  - Installing yiisoft/yii (1.1.14): Loading from cache
  - Installing jalle19/yii-lazy-image (1.0.2): Loading from cache
  - Installing mobiledetect/mobiledetectlib (2.7.9): Loading from cache
  - Installing mptre/php-xspf (v2.0.1): Downloading (failed)    Failed to downlo
ad mptre/php-xspf from dist: The "https://api.github.com/repos/mptre/php-xspf/zi
pball/29a61b26e5e2d52502a39d32a063b831612b44de" file could not be downloaded (HT
TP/1.1 404 Not Found)
    Now trying to download from source
  - Installing mptre/php-xspf (v2.0.1): Cloning 29a61b26e5


  [RuntimeException]
  Failed to clone https://github.com/mptre/php-xspf.git via https, ssh protoc
  ols, aborting.
  - https://github.com/mptre/php-xspf.git
    Cloning into 'C:\xampp\htdocs\xbmc-video-server\vendor\mptre\php-xspf'...
    remote: Repository not found.
    fatal: repository 'https://github.com/mptre/php-xspf.git/' not found
  - git@github.com:mptre/php-xspf.git
    Cloning into 'C:\xampp\htdocs\xbmc-video-server\vendor\mptre\php-xspf'...
    Host key verification failed.
    fatal: Could not read from remote repository.

    Please make sure you have the correct access rights
    and the repository exists.

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-c
ustom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest
] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative]
 [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...

====================================
```

**Comment by Jalle19** [comment]
Strange, one of the libraries seems to have been removed from the internet completely. I'll have to come up with a replacement. Thanks for reporting this!

**Comment by Jalle19** [comment]
Fixed in v1.8.4. Just pull the latest code and the `composer install` step should work again.
